src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -18,7 +18,7 @@ namespace System
     /// The methods use hardware intrinsics when available on the underlying platform,
     /// otherwise they use optimized software fallbacks.
     /// </summary>
-    internal static class BitOps
+    public static class BitOps
```

Comment:
It was approved in https://github.com/dotnet/corefx/issues/35419 with a different name and a different namespace.

Reply:
Yes - WIP, I need to update various callsites

Reply:
Added WIP to title

Reply:
OK, I think it's all done, let me know if I missed anything.